### ISSUE TITLE
Automatically activate Go versions based on per-directory .go-version files

### ISSUE THREAD

**Issue by mcandre** [moovweb/gvm#256]
Could gvm operate like rvm and nvm, so that `cd`ing to a directory automatically `gvm use`s the appropriate Go version? Developers working on projects with many different Go versions could document and manage version switching with simple `.go-version` files.

**Comment by markeissler** [comment]
Yes, there's already a PR for this (https://github.com/moovweb/gvm/pull/233) but it still has to be merged (ahem: @jbussdieker).

**Comment by markeissler** [comment]
@mcandre [GVM2](https://github.com/markeissler/gvm2) is now in its first release. Go check it out.
